books/views.py

```diff
@@ -184,3 +185,11 @@ def user_page_widget(request, username):
     user = get_object_or_404(User, username=username)
     books = UserBook.objects.select_related('book').filter(user=user, status='c')
     return render(request, 'widget.html', {'books': books})
+
+
+def user_favorite(request):
+    user = request.GET.get('user')
```

Comment:
this does not seem secure, best to get the user from the session (`request` object), probably want to protect this endpoint using the `@login_required` decorator

Reply:
My bad, It was initially in my scope but I guess totally forgot it. Will fix this